### PR TITLE
TestLodge integration

### PR DESCRIPTION
**Pull request by scottsherwood**
Adding TestLodge integration.

### CODE REVIEW

File: integrations.yml [hunk 1]
```diff
@@ -58,3 +58,9 @@
   website: "https://eazybi.com/integrations/basecamp"
   icon: "eazybi.png"
   category: 3
+-
+  name: "TestLodge"
+  description: "A test management tool that will create to-do's whenever a test fails allowing you to use Basecamp as an issue tracker."
```
Comment (javan) [inline integrations.yml]: *to-dos (not to-do's)

**Comment by javan** [comment]
Thanks! Just to confirm, your integration works with Basecamp 3, right? I wasn't able to confirm from your website.

**Comment by scottsherwood** [comment]
@javan Yes, the integration works with both versions 3 and 2 now. The option to select the correct version is shown when setting up the integration.

**Comment by javan** [comment]
Great, thanks. Mind fixing up that typo in the description?

**Comment by scottsherwood** [comment]
@javan Sure, just committed, thanks.

**Pull request merged**